cve_json_schema/v5.x_discuss/cve508_containerized.schema

```diff
@@ -505,7 +533,8 @@
                                 "CWE-ID": {
                                     "type": "string",
                                     "description": "CWE ID of the CWE that best describes this problem_type entry",
-                                    "minLength": 1
+                                    "minLength": 5,
+                                    "pattern":"^CWE-[0-9]+$" 
```

Comment:
Technically this allows an invalid `CWE-0`, could use this instead: `"^CWE-[1-9]\\d{0,5}$"`

Reply:
What is the CWE for "No problem"? ;-)

Conceptually, a CNA could add a ADP container with CWE set to CWE-0 to provide an opinion that the CVE is a non-issue.

Reply:
I don't think `CWE-0` is the right way to indicate "no security impact" :-) If a CNA really wanted to do that (as opposed to *not* publishing a non-vulnerability in the first place), they could assign it a CVSS score of 0.0.